build/openwisp_openvpn/openvpn.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/sh
+
+# This script will be called by cronjob to
+# update OpenVPN configurations periodically.
+source /utils.sh
+
+init_conf
```

Comment:
Does this need/is it safe to run init_conf periodically?  I assume this is to capture a TZ or Database password change?

Reply:
It's safe to run, It's only settings the PSQL default variables, but I can see this being a bit confusing in future, in case someone adds something to `init_conf` which isn't safe to run again.

So, I'll make psql a seperate function and make it more clear by adding a comment! :smile:

Reply:
Seperated the default psql variables in a new function now! :smile: